tests/systemtest/test_post_crash.py

```diff
@@ -27,22 +27,22 @@ def add_error(self, msg):
         self.errors.append(msg)
 
     def raw_crash_key(self, crash_id):
-        return '/v2/raw_crash/{entropy}/{date}/{crashid}'.format(
+        return 'v2/raw_crash/{entropy}/{date}/{crashid}'.format(
```

Comment:
The `/` at the beginning would cause the full pseudo-filename to have a `//` in it which is a different path than what Antenna is saving to.

I fixed this issue in Antenna in https://github.com/mozilla/antenna/commit/7c6637a826c4e2b7ab6adff80d776f9ecee71863 but must not have carried it through to the system tests. My bad. :(